internal/provider/providertest/utils.go

```diff
@@ -23,6 +25,21 @@ var setupMutex sync.Mutex
 var isAccountCreated bool
 var testConfigCache = make(map[string]string, 0)
 
+// IDRegex expression for Pipeline IDs
+var IDRegex = regexp.MustCompile(`[\w-]{36}`)
+
+// ParsedAccConfig is a helper function to parse the account configuration
```

Comment:
This is why I thought it was "account".  I won't plant a flag on naming it fully, even though I hate abbreviations.  If you change this commit to read "acceptance" then you can keep the name.  Maybe something like: 
```
Receives an HCL template and substitutes the `config` values for use in acceptance tests.
```

Reply:
Oh good catch, I did not see the account typo in the comment 😵‍💫